modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -83,18 +83,18 @@ spec:
      outputRefs:
      - secureforward-offcluster
 ----
-<1> The name of the log forwarding CR must be `instance`.
-<2> The namespace for the log forwarding CR must be `openshift-logging`.
-<3> Set to `enabled` to enable log forwarding.
+<1> The name of the Log Forwarding CR must be `instance`.
+<2> The Namespace for the Log Forwarding CR must be `openshift-logging`.
```

Comment:
This one is fun, because [Namespace](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#namespace-v1-core) is a k8s resource, but it's also a field in every single CR, I think without exception, and references a Namespace object.

Here we mean the `namespace` field, though. So I'd say:

The `namespace` for the Log Forwarding CR must be `openshift-logging`.